src/components/Card.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import '../styles/card.scss';
+
+export const Card = ({ card, onClick, isFlipped, inactive }) => {
```

Comment:
👍 This is awesome. Now card is a what we called a "dummy component" or "presentation component". What it rendered purely depends on input props thus makes this component very clean and therefore "dumb".

This is a very important concept while designing components. What should a component know. Where should this logic be. It's okay to be messy or you can put everything into one big component first but always remember to move repetitive/reasonable logic out as an individual component.